Composer/plugins/samples/assets/projects/CustomAction/schemas/MultiplyDialog.schema

```diff
@@ -0,0 +1,25 @@
+﻿{
```

Comment:
This is unnecessary, and doing the schema here and doing the code in another place actually make the CSharp Bot template not "self-contained".